lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -40,7 +40,9 @@
                 required: true,
                 editable: 'always',
                 showMessages: false,
-                title: gettext('edX learners can see my:'),
+                title: interpolate_text(
+                    gettext('{platform_name} learners can see my:'), {platform_name: options.platform_name}
```

Comment:
@andy-armstrong why is this a problem for right-to-left? I'm fine with long platform names.